openid-4-verifiable-presentations-1_0.md

```diff
@@ -222,6 +222,8 @@ Figure: Cross Device Flow
 OpenID for Verifiable Presentations extends existing OAuth 2.0 mechanisms as following:
 
 * A new `presentation_definition` Authorization Request parameter that uses the [@!DIF.PresentationExchange] syntax is defined to request presentation of Verifiable Credentials in arbitrary formats. See (#vp_token_request) for more details. 
+* A new query language is defined to enable requesting Verifiable Credentials in a more flexible way. See (#vp_query) for more details.
+* A new `vp_query` Authorization Request parameter is defined to request presentation of Verifiable Credentials in the JSON-encoded VP Query format. See (#vp_token_request) for more details.
```

Comment:
I know its only bikeshedding but I wonder if `credential_query` is a more term accurate, the way I read this in plain english is that vp is the subject of the query, when in reality its not, the credential is, we could also go more generic and maybe just have query?

Reply:
I think I prefer the symmetry between `vp_query` and `vp_token`.

Reply:
`vc_query` and `vp_token`?

Reply:
maybe if we have a name for this query language, we could use the abbrevication here?

Reply:
`dcql_query`